README.md

```diff
@@ -49,4 +49,5 @@ The only external finance system currently supported is E5.
 Pull image from private CH registry by running `docker pull 169942020521.dkr.ecr.eu-west-2.amazonaws.com/local/lfp-pay-api:latest` command or run the following steps to build image locally:
 
 1. `export SSH_PRIVATE_KEY_PASSPHRASE='[your SSH key passhprase goes here]'` (optional, set only if SSH key is passphrase protected)
+# TODO: this command is to be revisited when we know more about the AWS setup
```

Comment:
I think we can remove this TODO as currently the latest image of `lfp-pay-api` is what we have in eu-west-2.

Reply:
Will remove.